superset-frontend/src/SqlLab/components/TableElement.jsx

```diff
@@ -156,26 +137,28 @@ class TableElement extends React.PureComponent {
         {keyLink}
         <IconTooltip
           className={
-            `fa fa-sort-${!this.state.sortColumns ? 'alpha' : 'numeric'}-asc ` +
+            `fa fa-sort-${!sortColumns ? 'alpha' : 'numeric'}-asc ` +
```

Comment:
small nit, and this is from earlier code, but for readability: 
``fa fa-sort-${sortColumns ?  'numeric' : 'alpha' }-asc ` +`